.pre-commit-config.yaml

```diff
@@ -1,25 +1,10 @@
 repos:
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.12.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.2.1
     hooks:
-      - id: isort
-  - repo: https://github.com/psf/black
-    rev: 23.11.0
-    hooks:
-      - id: black
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
-    hooks:
-      - id: prettier
-        exclude: ^tests/fixtures/
```

Comment:
Ruff does not replace Prettier, so we should keep this hook.

Reply:
Made the necessary changes. I see you have also made some changes. Let me know if I was wrong somewhere. Thank you for your guidance in this endeavor